code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -73,21 +73,30 @@
 	..()
 	gear_tweaks += gear_tweak_free_color_choice
 
-/datum/gear/uniform/qipao
-	display_name = "qipao"
-	path = /obj/item/clothing/under/qipao
+/datum/gear/uniform/qipao_colorable
+	display_name = "qipao, selection (colorable)"
+	path = /obj/item/clothing/under/qipao/colorable
 
-/datum/gear/uniform/qipao/New()
+/datum/gear/uniform/qipao_colorable/New()
 	..()
-	gear_tweaks += gear_tweak_free_color_choice
+	var/paotype = list()
+	paotype["qipao, gold trim"] = /obj/item/clothing/under/qipao/colorable
```

Comment:
Can you make the list initialization consistent?

Reply:
Switched all the 'static' lists to the more readable var/list/things = list( "thing 1" = path,  "thing 2" = path) rather than the older var/thinglist thinglist = list thingtype["thing 1"] = path builder-me-bob.
(Sub)type-generated lists are still more similar to the builder-me-bob style wise but I'm not sure if theres a Huge Advantage to it being one way or another besides the construction lists being slightly more faff to add in future. Could go back and builder-me-bob-ify everything instead if its secretly way better not to be static but SHRUGS